compiler/modules/print.llvm/tests/testOutputs/addr_space_cast.ll

```diff
@@ -0,0 +1,6 @@
+declare i64 addrspace (3 )* @genFn ()
+define i64* @main () {
+  %_0 = call i64 addrspace (3 )* @genFn ()
+  %_1 = addrspacecast i64 addrspace (3 )* %_0 to i64*
```

Comment:
Can we remove this extra space here? 
`addrspace (3 )*` -> `addrspace (3)*`

Reply:
Fixed